types/types.go

```diff
@@ -67,6 +69,27 @@ type memMarker struct {
 	mtx sync.RWMutex
 }
 
+// Count alerts of a given state.
+func (m *memMarker) Count(states ...AlertState) int {
+	count := 0
+
+	m.mtx.RLock()
+	defer m.mtx.Unlock()
```

Comment:
This should be `RUnlock`, when using `RLock`.

Reply:
True, sorry for that, fixed